daq/faucet_events.py

```diff
@@ -0,0 +1,53 @@
+from collections import namedtuple
+
+EVENT_TYPE_PORT_STATE = "Port State"
+EVENT_TYPE_PORT_LEARN = "Port Learn"
+EVENT_TYPE_CONFIG_CHANGE = "Config Change"
+
+all_events = {}
```

Comment:
these aren't events at this point (in the varaible), it's system state that's being updated from the events. so 1) name should be reflective of the meaning, and 2) likely the name of the file is off too -- it's really a "state_collector" .. and should be a class not a collection of static methods